TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -134,19 +126,20 @@ formats in `R`.
 ## Exploring Date and Time Classes
 
 ###R - Date Class - as.Date
-We can use the`as.Date` method to convert a date field in string/character 
-format to an `R` `Date` class. It will ignore all values after the date string.
+As we just saw, when we use the`as.Date` method to convert a date field in string/character 
+format to an `R` `date` class, it will ignore all values after the date string.
+Let's explore date and time classes in more detail separately from our data set.
 
 ```{r explore-as_date }
 #Convert character data to date (no time) 
 myDate <- as.Date("2015-10-19 10:15")   
 str(myDate)
-head(myDate)
+myDate
```

Comment:
AHHHH -- gotcha. I need to run the code. So feel free to leave it if it's just one point. It's just nice to generally use head so to avoid resource issues and for pretty knitting :)